examples/csharp/HelloworldUWP/GreeterClient/Pregenerated Client Stubs/Greet.cs

```diff
@@ -0,0 +1,310 @@
+// <auto-generated>
```

Comment:
you can use Grpc.Tools nuget package to autogenerate the Greet.cs and GreetGrpc.cs, so you don't need to check them in (see other examples)

Reply:
Indeed the files can be generated. But then they can not be compiled as part of the client. The current XAML/C# compilation pipeline empties the generated files if they are part of the project (referred in the .csproj file), and ignores them if they're not. So for a HelloWorld project I assume it would be better to just add the pregenerated files (as in the Xamarin sample).

Reply:
I added client stub generation, for reference. The generated files are not used to compile.